requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.22.0
 scipy>=1.5.0,<=1.11.2
 cirq>=1.0.0,<1.3.0
+tabulate
```

Comment:
I feel a bit uncomfortable to have a new line in `requirements.txt`. 
But maybe it's fine given that is a small package.

Reply:
I don't love it either, but you're right it is a small package. I think it's worth adding to avoid the pain in trying to create these tables ourselves and maintaining that code, however.

There is the option to make `tabulate` and "extra" package which could be installed via `pip install mitiq[calibration]` by moving this requirement out of here and into another file. We already do this with our development dependencies which are then passed into the `extra_require` argument in the `setup` function:
https://github.com/unitaryfund/mitiq/blob/cc7a3e1e1567531ac975762030a9c6eac1537a71/setup.py#L29-L31

WDYT? I have a slight preference to leave as is, since we want calibration to (eventually) be one of the main ways people use mitiq and requiring additional steps for a happy path isn't quite in line with that. That said, totally understand that adding more main requirements can be a burden.

Reply:
fine to keep it as is.